### PR TITLE
RDISCROWD-3403: Integrating multiselect component

### PR DESCRIPTION
**Pull request by ciaracian**
*Issue number of the reported bug or feature request:*
https://jira.prod.bloomberg.com/browse/RDISCROWD-3403

**Describe your changes**
I added the Multiselect component to the code builder. This required creating the MultiselectForm.vue and multiselectTemplate.html, as well as updating the required index, router, etc. files. 

**Testing performed**
I added the required tests which are passing.

**Additional context**
In order to create the new files I looked at the DropdownInput component and replicated most of the logic. The primary difference between the two components is that MultiselectInput will only accommodate lists for the "choices" while DropdownInput can use lists or dicts.

### CODE REVIEW

File: static/src/components/builder/components/MultiselectInput/MultiselectForm.vue [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+      :validations="validations"
+      :validation-options="filteredValidations"
+      :update-validations="updateValidations"
+    />
+    <div class="form-group">
+      <label
+        class="col-labels"
+        for="initial-value"
+      >
+        Initial Value | <span class="label-tip">Must be one of the configured multiselect values.</span>
```
Comment (ywrsusan) [inline static/src/components/builder/components/MultiselectInput/MultiselectForm.vue:41]: Is the initial value field a single-select dropdown or a multi-select dropdown?

File: static/src/components/builder/components/ComponentRender/ComponentRender.vue [hunk 2]
```diff
@@ -112,6 +112,15 @@ export default {
           props: {
             pybAnswer: this.form.pybAnswer,
             choices: this.form.choices,
+            initialValue: this.form.initialValue[0]
```
[the remark this answers is not in the document]
Comment (ciaracian) [reply static/src/components/builder/components/ComponentRender/ComponentRender.vue:115]: This was leftover from a previous implementation and no longer necessary. Thanks

File: static/src/components/builder/components/MultiselectInput/MultiselectForm.vue (hunk 1 above)
Comment (ciaracian) [reply static/src/components/builder/components/MultiselectInput/MultiselectForm.vue:41]: I don't really understand the question. I will follow up via IB!

File: static/src/components/builder/store/modules/multiselectInput.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        else uniqueValues.add(trimmedValue);
+    }
+}
+
+export const getters = {
+  [types.GET_MULTISELECT_INPUT_PROPS] (state) {
+    return {
+      label: state.label,
+      labelAdded: state.labelAdded,
+      choices: state.choiceList.map((choice) => { return choice.value.trim(); }),
```
Comment (XiChenn) [inline static/src/components/builder/store/modules/multiselectInput.js:43]: For ES6 arrow function, curly braces and 'return' keyword are redundant for single return statement. The bracket for single parameter can also be omitted. If you like, you can write this line as choices: state.choiceList.map(choice => choice.value.trim())
This comment applies to multiple places in the pr :)

Comment (ciaracian) [reply static/src/components/builder/store/modules/multiselectInput.js:43]: Ok, thanks Xi! I will fix these.

**Review by XiChenn — APPROVED** [review]
lgtm

**Review by kbecker42 — APPROVED** [review]
Thank you.

**Pull request merged**